fix: adding missing chromiumPort parameter in web-ext.ts to enable fixed port debugging

sylvaneau opened the pull request:
This fix enable to setup a fixed port to enable easy debugging with VSCode or other IDEs

### Overview

Everything was already there, I've just added the line to pass the parameters sets in wxtUserConfig to the web-ext runner.

### Manual Testing

* Set `chromiumPort` in wxt.config.ts.
* Launch `npm run dev`
* Browse to chrome://version
* Check that `remote-debugging-port` is the same as set in config file

You can also setup a fully working environment with this repo https://github.com/sylvaneau/wxt-react-vscode-template and try to debug your project with VSCode.

### Related Issue

<!-- If this PR is related to an issue, please link it here -->

This PR closes #52

sylvaneau commented:
> > I do not undernstand the failing checks, it seems it has no relation with the part i've modified.
> 
> Check failed because of PR title there's some kind of linter for it, now should be good, but it need to be approved by owner again.
> 
> I don't see all CI workflows because i'm on mobile currently 🙂

Hello, no I have already updated the name of the PR, the error is related to a completely different item:

```
src/virtual/utils/reload-content-scripts.ts(40,51): error TS2322: Type '{ id: string; matches: string[]; js?: string[]; css?: string[]; }' is not assignable to type 'RegisteredContentScript'.
```

PatrykKuniczak commented:
@sylvaneau Yeah, PR title validation is valid right now, you need to fix issue with eslint in code, let's take deeper look on failed workflow

PatrykKuniczak commented:
@sylvaneau Ok, in that case it need to be fix in other PR.
Good job :)

sylvaneau commented:
Victory !!! \o/

However I have to admit it is not very straight forward...

So first I've updated my `.vscode/launch.json` sourcemap paths:

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Attach to Chrome",
            "port": 9229,
            "request": "attach",
            "type": "chrome",
            "webRoot": "${workspaceFolder}/src",
            "sourceMaps": true,
            "sourceMapPathOverrides": {
                "../../../src/*": "${webRoot}/*",
                "../../src/*": "${webRoot}/*",
            }
        },
    ]
}
```

(btw I've updated my template project with this change)

Once you've done that you have to respect a particular sequence to let VSCode to be aware of the background script.

1. run `npm run dev`
2. on chrome click on your extension icon to display the popup / right click -> 'Inspect'
3. go to the 'Application' tab, select 'Service Workers' on the left panel
4. you will see your background script, next to the 'Status' line click the 'Stop' button
5. now go back to VSCode and hit F5 to attach debugger
6. go back to Chrome, click the 'Start' button and "Voilà" ! you can set breakpoints !

<img width="700" height="262" alt="background_script_debug" src="https://github.com/user-attachments/assets/4f743707-bbb0-4ab8-a35b-e2dadd6feb61" />

If you attach VSCode before stopping the service worker it will not work...

aklinker1 commented:
Don't use VSCode personally, but this is a good fix regardless. Merged with latest `main` to fix checks.
